int32 state
int32 remainTime
int32 secondTime
int32 redScore
int32 blueScore

int32 STATE_INIT = 0
int32 STATE_READY = 1
int32 STATE_PLAY = 2
int32 STATE_PAUSE = 3
int32 STATE_END = 4
